.travis.yml

```diff
@@ -46,7 +46,7 @@ env:
 before_script:
   - sudo iptables -L DOCKER || ( echo "DOCKER iptables chain missing" ; sudo iptables -N DOCKER )
 script:
-  - bundle exec $CMD $OPTIONS
+  - travis_wait 100 bundle exec $CMD $OPTIONS
```

Comment:
can you explain why you put a `travis_wait` here?

Reply:
Im not sure, this was a long time ago. But it looks like it was in the second commit that got travis fixed.

I can try removing this and see if travis runs my tests.